miqa/learning/nn_classifier.py

```diff
@@ -17,6 +17,10 @@
 from torch.utils.tensorboard import SummaryWriter
 import wandb
 
+log_level = os.environ.get('LOGLEVEL', 'WARNING').upper()
+logging.basicConfig(level=log_level)
```

Comment:
I think we should move the above two lines into the `__main__` block, for whoever is using this from the command line.